src/features/ci-cd/hooks/backfill-device-service-environment-variable.ts

```diff
@@ -0,0 +1,35 @@
+import { hooks, errors, type sbvrUtils } from '@balena/pinejs';
```

Comment:
Moving this to eg a `service-install` feature would make more sense to me as it doesn't actually relate to ci-cd really

Reply:
done